### ISSUE TITLE
Modded dimension sky

### ISSUE THREAD

**Issue by BlueAmulet** [Sildurs-shaders/sildurs-shaders.github.io#124]
Since the overworld shaders are the default shaders, any modded dimension that has it's own specific sky color ends up rendering with an overworld sky, which can end up clashing a bit. A good example of this is the Extra Utilities 2's Deep Dark, where the sky is supposed to be black along with the black fog, and yet the sky is a bright blue.

I've partially solved this myself by duplicating all the shaders to a world0 folder and then trying to edit the default shader sky to use the skyColor uniform, but apparently this is the raw sky color and doesn't take into account the time of day?

**Comment by Sildurs-shaders** [comment]
There's a default skybox option in the shader settings, maybe that works

**Comment by BlueAmulet** [comment]
None of this is working properly for any dimension I've tested. Default Skybox doesn't seem to fix the sky color, just makes a very tiny difference, but I left it on for the testing. In the dimension I've tested (Deep Dark, The Beneath, The Erebus, Twilight Forest, The Abyss), depending on what time of day you enter the dimension determines how the sky will act. Entering during day make the sky go between a bright blue (day) and a lavendar (night), and entering during night makes the sky stay forever dark blue regardless of time. I'm also unsure what the Skybox Blend Factor is doing? Setting it to 0 just makes the sky solid black in all dimensions, and setting to 1 just makes the sky the wrong blue/lavendar combo.

For reference this was tested again:
Sildurs Vibrant Shaders v1.282 Extreme

Minecraft 1.12.2
Forge 14.23.5.2847
preview_OptiFine_1.12.2_HD_U_G5_pre1

Minecraft 1.16.4
Forge 35.1.37
OptiFine_1.16.4_HD_U_G6
